app/components/TextWithIcon/index.tsx

```diff
@@ -45,7 +45,9 @@ const TextWithIcon = ({
       ) : (
         <>{icon}</>
       )}
-      <div>{iconRight ? <></> : <>{content}</>}</div>
+      <div className={styles.overflowWrap}>
+        {iconRight ? <></> : <>{content}</>}
+      </div>
```

Comment:
Why do we want an empty div if the icon is on the right side? I'm sure there's a reason though😆 (so check that out before removing hehe)